examples/main.tf

```diff
@@ -1,8 +1,29 @@
 terraform {
   required_providers {
     massdriver = {
-      version = "0.0.1"
-      source  = "massdriver.cloud/massdriver"
+      version = "~> 1.0.0"
+      source  = "massdriver-cloud/massdriver"
+    }
+  }
+}
+
+variable md_name_prefix {
```

Comment:
This might encourage people to create their own variable declarations, which we don't want them to do.